plugin/trino-hudi/src/test/java/io/trino/plugin/hudi/TestHudiOnHive3.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.hudi;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import io.trino.plugin.hive.containers.HiveMinioDataLake;
+import io.trino.plugin.hudi.testing.TpchHudiTablesInitializer;
+import io.trino.testing.AbstractTestQueryFramework;
+import io.trino.testing.QueryRunner;
+import org.testng.annotations.Test;
+
+import static io.trino.plugin.hive.containers.HiveHadoop.HIVE3_IMAGE;
+import static io.trino.plugin.hudi.S3HudiQueryRunner.HUDI_MINIO_TESTS;
+import static io.trino.testing.sql.TestTable.randomTableSuffix;
+import static org.apache.hudi.common.model.HoodieTableType.COPY_ON_WRITE;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TestHudiOnHive3
+        extends AbstractTestQueryFramework
+{
+    @Override
+    protected QueryRunner createQueryRunner()
+            throws Exception
+    {
+        String bucketName = "test-hudi-connector-" + randomTableSuffix();
+        HiveMinioDataLake hiveMinioDataLake = closeAfterClass(new HiveMinioDataLake(bucketName, HIVE3_IMAGE));
+        hiveMinioDataLake.start();
+        hiveMinioDataLake.getMinioClient().ensureBucketExists(bucketName);
```

Comment:
`HiveMinioDataLake.start` does that, doesn't it?
https://github.com/trinodb/trino/blob/e73567b47412105206c9272e8edc95f98c771a4f/plugin/trino-hive/src/test/java/io/trino/plugin/hive/containers/HiveMinioDataLake.java#L172